packages/jest-runtime/src/cli/index.ts

```diff
@@ -82,7 +82,10 @@ export function run(cliArgv?: Config.Argv, cliInfo?: Array<string>) {
     watchman: globalConfig.watchman,
   }) as Promise<Context>)
     .then(hasteMap => {
-      const Environment: typeof JestEnvironment = require(config.testEnvironment);
+      const transformer = new Runtime.ScriptTransformer(config);
+      const Environment: typeof JestEnvironment = interopRequireDefault(
+        transformer.requireAndTranspileModule(module, config.testEnvironment),
```

Comment:
This seems to not work when I call it as `transformer.requireAndTranspileModule`, but works fine if I replace it with the same code inside `requireAndTranspileModule`.

This is the reason the tests are failing now, I'm looking into fixing it